tests/components/miele/test_init.py

```diff
@@ -157,3 +159,39 @@ async def test_device_remove_devices(
         old_device_entry.id, mock_config_entry.entry_id
     )
     assert response["success"]
+
+
+async def test_api_push(
+    hass: HomeAssistant,
+    mock_miele_client: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    device_registry: dr.DeviceRegistry,
+    device_fixture: MieleDevices,
+    action_fixture: MieleAction,
+) -> None:
+    """Test device registry integration."""
+    await setup_integration(hass, mock_config_entry)
+    device_entry = device_registry.async_get_device(
+        identifiers={(DOMAIN, "Dummy_Appliance_1")}
+    )
+    assert device_entry is not None
+
+    entity_id = "climate.refrigerator"
+    state = hass.states.get(entity_id)
+    assert state
+    assert state.attributes.get(ATTR_MIN_TEMP) == -28
+    assert state.attributes.get(ATTR_MAX_TEMP) == 28
+
+    await MieleDataUpdateCoordinator.callback_update_data(
+        mock_config_entry.runtime_data, device_fixture
+    )
```

Comment:
We don't want to call it directly like this, we want to use the method passed to the library

Reply:
I understand, but I don't see how that can be done with the current design. This way we at least verify that data arriving by push is handled correctly.
Or have you got a suggestion? I would rather try to avoid a major redesign of the push/callback structure. It took quite some time to get in place.

Reply:
Your code registers a callback to the library. We can use the call_args of the mock to find that function reference and call it in the tests

Reply:
I think I got it now.